fiasco/ion.py

```diff
@@ -290,7 +292,8 @@ def level_populations(self,
         try:
             _ = self._psplups
         except KeyError:
-            # TODO: log this
+            warnings.warn('psplups data not available, not including protons '
```

Comment:
I'm now wondering whether this is the right approach, or if this should just be removed and an error thrown in the `if include_protons` block by just putting the `proton_collision_excitation_rate` call first. This of course means if you're iterating over a number of ions, you're constantly having to catch this exception so maybe this is best how it is.

Reply:
I'll get rid of this for now, I think it snuck in as I wanted to double check what files I needed to add.